### PR TITLE
WordParser (Docx files) Automation

### PR DESCRIPTION
**Pull request by ilaif**
[ADD] WordParser script that given an entryID in docx, parses it as txt.
[ADD] Test helper script to create a docx file given input.
[ADD] Test playbook for WordParser (and to conf.json).
* Added docker image for the automation named demisto/word-parser.

### CODE REVIEW

**Comment by ilaif** [comment]
@anara123

File: Scripts/script-WordParser.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  except PackageNotFoundError as e:
+      errEntry["Contents"] = "Input file is not a valid docx/doc file."
+      res = errEntry
+  except BaseException as e:
+      errEntry["Contents"] = "Error occurred while parsing input file.\nException info: " + str(e)
+      res = errEntry
+
+  demisto.results(res)
+type: python
+tags: []
```
Comment (meirwah) [inline Scripts/script-WordParser.yml]: add tags like word, parser

**Comment by meirwah** [comment]
@ilaif you need to resolve conflicts

**Comment by anara123** [comment]
@ilaif Do you published already the docker image `demisto/word-parser`?

**Comment by meirwah** [comment]
@anara123 yes: https://hub.docker.com/r/demisto/word-parser/

File: Scripts/script-WordParser.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+commonfields:
+  id: WordParser
+  version: -1
+name: WordParser
+comment: Takes an input docx file (entryID) as an input and saves an output text file (file entry) with the original file's contents.
+script: |
+  from docx import Document
+  from docx.opc.exceptions import PackageNotFoundError
+
+  max_file_size = 1024 ** 2
```
Comment (orius123) [inline Scripts/script-WordParser.yml]: Where do you use it? why this number?

File: TestPlaybooks/script-TestCreateWordFile.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      "Type": entryTypes["error"],
+      "ContentsFormat": formats["text"],
+      "Contents": ""
+  }
+
+  args = demisto.args()
+  try:
+      document = Document()
+      document.add_paragraph(args['text'])
+      demisto.log('1')
```
Comment (orius123) [inline TestPlaybooks/script-TestCreateWordFile.yml]: remove this logs or make them meaningful

File: Scripts/script-WordParser.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+commonfields:
+  id: WordParser
```
Comment (orius123) [inline Scripts/script-WordParser.yml]: @meirwah not sure about the name, maybe 
`ParseDocFile` ? `ParseWordFile`?

**Comment by orius123** [comment]
@ilaif looks GR8! resolve conflicts and handle those minor comments.

File: Scripts/script-WordParser.yml (hunk 2 above)
Comment (ilaif) [reply Scripts/script-WordParser.yml]: No reason, other attempts leftovers. Removed.

File: TestPlaybooks/script-TestCreateWordFile.yml (hunk 3 above)
Comment (ilaif) [reply TestPlaybooks/script-TestCreateWordFile.yml]: Roger that.

**Comment by anara123** [comment]
@ilaif I got this error for 2 different word files with .docx extension and their content is English only
image

**Pull request merged**